test/jdk/java/nio/file/Files/CopyAndMove.java

```diff
@@ -62,7 +62,7 @@ public static void main(String[] args) throws Exception {
             // Use test.dir to define second directory if possible as it might
             // be a different volume/file system and so improve test coverage.
             String testDir = System.getProperty("test.dir", ".");
-            Path dir2 = TestUtil.createTemporaryDirectory(testDir);
+            Path dir2 = TestUtil.createTemporaryDirectory(testDir).toAbsolutePath();
```

Comment:
This change looks okay, do you mind update the Oracle copyright header end date to 2023 too.